ui/nuxeo-layout.js

```diff
@@ -112,10 +112,7 @@ import './nuxeo-error.js';
     }
 
     // Trigger the layout validation if it exists.
-    validate() {
-      if (this.element && typeof this.element.validate === 'function') {
-        return this.element.validate();
-      }
+    async validate() {
```

Comment:
why is the method flagged as `async`?

Reply:
because `this.element.validate();` (https://github.com/nuxeo/nuxeo-elements/pull/183/files#diff-0d576b0b68172de9bc11d6609e92ec21R129) can potentially return a promise